alien-module/control.lua

```diff
@@ -173,7 +170,7 @@ script.on_event(defines.events.on_entity_died, function(event)
 				end
 			end
 			
-			pp("Alien Hyper Modules upgraded to level: " .. global.modulelevel)
+			pp("异星超级插件提升至: " .. global.modulelevel .. "级")
```

Comment:
Thanks for the translation but i think the english text would be preferable here